lib/ansible/module_utils/basic.py

```diff
@@ -516,6 +516,18 @@ def is_executable(path):
             or stat.S_IXOTH & os.stat(path)[stat.ST_MODE])
 
 
+class AnsibleFallbackNotFound(Exception):
```

Comment:
I wasn't entirely sure where to put this. If someone has a better idea, I'd love to hear it.